src/app/components/Modals/WithdrawV2Modal.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  DialogHeader,
+  Text,
+} from "@babylonlabs-io/bbn-core-ui";
+
+import { getNetworkConfig } from "@/config/network.config";
+interface UnbondModalProps {
+  open: boolean;
+  onClose: () => void;
+  onProceed: () => void;
+  awaitingWalletResponse: boolean;
+}
+
+const { networkName } = getNetworkConfig();
+
+export const WithdrawModalV2 = ({
+  open,
+  onClose,
+  onProceed,
+  awaitingWalletResponse,
+}: UnbondModalProps) => {
+  const title = "Withdraw";
+  const content = (
+    <>
+      You are about to withdraw your stake. <br /> A transaction fee will be
+      deduced from your stake by the {networkName} network
+    </>
+  );
+
+  return (
+    <Dialog open={open} onClose={onClose}>
+      <DialogHeader className="text-primary-main" onClose={onClose}>
+        {title}
+      </DialogHeader>
+      <DialogBody className="pb-8 pt-4 text-primary-dark">
+        <Text variant="body1">{content}</Text>
+      </DialogBody>
+      <DialogFooter className="flex gap-4">
+        <Button
+          variant="outlined"
+          color="primary"
+          onClick={onClose}
+          className="flex-1"
+        >
+          Cancel
+        </Button>
+        <Button variant="contained" onClick={onProceed} className="flex-1">
+          {awaitingWalletResponse ? (
```

Comment:
I suggest checking in with the design about `awaitingWalletResponse`. Should we see `X` and `Cancel` if we are awaiting the wallet response?
https://i.imgur.com/j2pSspB.png

I.e. are we okay clicking "Proceed", seeing the Chrome Extension wallet popup, then closing the Withdrawal modal

^ Above does not mean that we should change something, maybe it is good as it is now